tests/integration/standard/test_query.py

```diff
@@ -942,7 +942,7 @@ def test_no_connection_refused_on_timeout(self):
                 exception_type = type(result).__name__
                 if exception_type == "NoHostAvailable":
                     self.fail("PYTHON-91: Disconnected from Cassandra: %s" % result.message)
-                if exception_type in ["WriteTimeout", "WriteFailure", "ReadTimeout", "ReadFailure", "ErrorMessageSub"]:
+                if exception_type in ["WriteTimeout", "WriteFailure", "ReadTimeout", "ReadFailure", "ErrorMessageSub", "ErrorMessage"]:
```

Comment:
Do we know why we're suddenly seeing the parent class `ErrorMessage` here when we weren't before?

Reply:
Driver receives exceptions:
```
WriteTimeout('Error from server: code=1100 [Coordinator node timed out waiting for replica nodes\' responses] message="CAS operation timed out - encountered contentions: 18" info={\'consistency\': \'SERIAL\', \'required_responses\': 2, \'received_responses\': 0, \'write_type\': \'CAS\'}')

ErrorMessage: <Error from server: code=1700 [Unknown] message="CAS operation result is unknown - proposal accepted by 1 but not a quorum.">
```

It looks clear [here](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/transport/messages/ErrorMessage.java#L192) why `CAS_WRITE_UNKNOWN` is wrapped in `ErrorMessage `. Cannot answer why this was not the case before.

I have asked HCD engineers around, but they are not familiar with LWT changes introduced in C* 5.0.

Reply:
Thanks for checking into it.  For the record I don't think this difference should hold up getting this PR in.  It seemed like an odd change when looking this over but I can envision perfectly good reasons why it might be the case.